.github/workflows/test-workflow.yml

```diff
@@ -47,7 +50,7 @@ jobs:
           version: nightly
 
       - name: Start the local node
-        run: npx hedera start -d --network local
+        run: npx hedera start -d --network-tag=${{inputs.networkTag}} --mirror-tag=${{inputs.mirrorTag}} --relay-tag=${{inputs.relayTag}} --verbose=trace
```

Comment:
Q: is the `--network local` no longer needed? Maybe local is the default?

Reply:
So local was always default, not sure why we explicitly used `--network local`. Current behaviour is you can use `--network local`, you can pass tags as we do it now, and if you pass empty strings instead, like `--network-tag=""`, which will be the case when they run on a PR network will be local (as it was until now).